x-pack/plugins/case/server/routes/api/utils.ts

```diff
@@ -35,14 +35,15 @@ export const transformNewCase = ({
   createdDate: string;
   email?: string;
   full_name?: string;
-  newCase: CaseRequest;
+  newCase: CasePostRequest;
   username: string;
 }): CaseAttributes => ({
   closed_at: null,
   closed_by: null,
   created_at: createdDate,
   created_by: { email, full_name, username },
   external_service: null,
+  status: 'open',
   updated_at: null,
   updated_by: null,
   ...newCase,
```

Comment:
move `...newCase` as first item so we overwrite the default values just in case badness happens